### PR TITLE
Renew PrestaShop installer branding

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Renew PrestaShop installer branding, agreed by @matks . See https://github.com/PrestaShop/PrestaShop/issues/36615
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | [Build a .zip](https://www.prestashop-project.org/maintainers-guide/processes/release/prestashop/create-build/#6-build-the-zip-archive) and try to install it manually and check if the new styles are applied
| UI Tests          | https://github.com/florine2623/testing_pr/actions/runs/10090095398 ✅ https://github.com/florine2623/testing_pr/actions/runs/10266582533 ✅ 
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

**Review by marsaldev — DISMISSED** [review]
I rely on @kpodemski

**Comment by Progi1984** [comment]
Question : What are licenses of fonts in `install-dev/theme/fonts/` ? OFL ?

https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/compatible-licenses/#artwork

**Comment by tblivet** [comment]
> Question : What are licenses of fonts in `install-dev/theme/fonts/` ? OFL ?
> 
> https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/compatible-licenses/#artwork

For IBM Plex Sans : https://github.com/IBM/plex?tab=OFL-1.1-1-ov-file#readme
But for Prestafont i have no idea !

**Comment by Progi1984** [comment]
> > Question : What are licenses of fonts in `install-dev/theme/fonts/` ? OFL ?
> > 
> > https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/compatible-licenses/#artwork
> 
> For IBM Plex Sans : https://github.com/IBM/plex?tab=OFL-1.1-1-ov-file#readme
> But for Prestafont i have no idea !

The PrestaShop open source project is distributed under the OSL 3.0 license. All dependencies included and distributed with PrestaShop must be compatible with this license.

Dixit the devdocs

**Comment by kpodemski** [comment]
@Progi1984 and what are the conflicts between OFL and PrestaShop license in your opinion? I don't see any.

**Comment by Progi1984** [comment]
I look for the license for the PrestaFont. As @tblivet wants to include it in the project, the license must be compatible with OSL, like IBM Plex Sans in OFL. So if the PrestaFont license is OFL, no problem. Else it's a new problem of legal compliance.

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

**Review by florine2623 — COMMENTED** [review]
Hello @tblivet ,

The branding looks good to me ^^
But I'm having some issues during the install process.

When I fill my **Store information** form and then click Next, I'm redirected to the Previous step **Store Compatibility** : :x:

https://github.com/user-attachments/assets/c73f1fa4-f76d-4f71-93e6-62798a2052e2

When I print my information and then Cancel, I'm redirected to the page **Store information**, instead of staying on the **Store installation** confirmation page : :x:

https://github.com/user-attachments/assets/57391126-b22f-4477-b5e4-1429e70d66b1


Also, would it be possible to have the right under the title ?
Screenshot 2024-07-25 at 11 46 04


Waiting for your feedback :)
Thanks!

**Comment by tblivet** [comment]
Hi @florine2623,
I'm not able to reproduce the issue where you are redirected to the previous step. Can you re-check if you still have this behavior? As for the other two issues, they should be fixed now ✅

@Progi1984, @kpodemski regarding the PrestaFont license, I have asked the legal team if the font is compatible with the open-source project. They should get back to me in the middle of next week. In case it's not compatible, removing it won't be an issue because it's only used on `<h1>`

File: tools/build/Library/InstallUnpacker/content/installer.gif
Comment (jolelievre) [inline tools/build/Library/InstallUnpacker/content/installer.gif:1]: Much less funny 😅
There's no plan to integrate a more entertaining animation later?

This new GIF is really heavy 1.34Mo for such a reduced animation, the initial GIF was only 164Ko and as more entertaining, do you think there's at least a way to reduce/optimize this file size?

**Review by jolelievre — COMMENTED** [review]
All good for me except for the GIF which seems to be way too heavy for what it shows

File: install-dev/theme/fonts/IBMPlexSans-Bold.ttf
Comment (jolelievre) [inline install-dev/theme/fonts/IBMPlexSans-Bold.ttf:1]: One question about this one, I believe it is also included in the UI kit modifications you made in this PR https://github.com/PrestaShop/prestashop-ui-kit/pull/229 Can you not use the assets already in the UI kit instead of interesting this font manually again here

Especially since it seems you also integrated it in the Unpacker, so it seems like the same font is integrated in three different places which is not optimal But maybe there are technical constraints that force this?

File: tools/build/Library/InstallUnpacker/content/installer.gif
Comment (tblivet) [reply tools/build/Library/InstallUnpacker/content/installer.gif:1]: I agree with you, but the original GIF was 9 Mo. I have compressed it as much as I can.

File: install-dev/theme/fonts/IBMPlexSans-Bold.ttf
Comment (tblivet) [reply install-dev/theme/fonts/IBMPlexSans-Bold.ttf:1]: I think there are technical constraints that force this. Everything is located in different places, and each part seems separate/autonomous.

**Comment by matks** [comment]
> @Progi1984, @kpodemski regarding the PrestaFont license, I have asked the legal team if the font is compatible with the open-source project. They should get back to me in the middle of next week. In case it's not compatible, removing it won't be an issue because it's only used on `<h1>`

From my point of view, there are 2 options:

1. The PrestaFont is included in the project without specific notice. Then it becomes part of the content of the project and then it is included in the licensed content. It means the PrestaFont is licensed under OSL.
2. The PrestaFont is included in the project with a specific license for it. This license has to be compatible with OSL.

In both options, the PrestaFont is licensed with an open license. The question is which one.

File: tools/build/Library/InstallUnpacker/content/installer.gif
Comment (Hlavtox) [reply tools/build/Library/InstallUnpacker/content/installer.gif:1]: Cropped, redured color pallette, reduced FPS to half, compressed - 140 kB.

installer-ezgif com-crop

But still, such a bland, empty animation. 🙄

**Review by florine2623 — COMMENTED** [review]
Hello @tblivet ,

Thanks for the new commit !

I still reproduce the first bug. 

https://github.com/user-attachments/assets/a91efad8-a88a-44a6-9bd0-81b26e64be7b

I have found out that when you press on the Enter key instead of clicking on the button with the mouse, the behavior is different. The behavior is different with and without the reskin.
For example, in the video, in Step 4, I enter my information and I press Enter, it redirected my to Step 3. In this case, it should let me know that the form isn't completed. :x:

https://github.com/user-attachments/assets/c1388b47-ff0e-4372-86e2-5553f137d4da

There are many more cases when pressing on Enter key has a different behavior than clicking on Next button with the mouse.

The second bug is fixed ✅ 

https://github.com/user-attachments/assets/d63846c8-35e7-43ce-9470-9b341fc79e80

The 3rd bug is fixed ✅ 
Screenshot 2024-08-06 at 16 14 17

Could you check, if you want we can check directly together.

Thanks!

**Review by florine2623 — COMMENTED** [review]
Thanks @tblivet ,

I have found a little something 😅 

The drop downs for `Country` and `Store timezone` are not well displayed.

Screenshot 2024-08-07 at 10 56 43

Screenshot 2024-08-07 at 10 58 15

The rest looks good to me !
Thank you :D

**Review by jolelievre — DISMISSED** [review]
Thanks @tblivet 

All good for me, except for the GIF that we are still waiting, it can be replaced in another PR but we mustn't forget about it

**Review by jolelievre — APPROVED** [review]
Thanks @tblivet

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
@tblivet

The latest issue is fixed :) 

Screenshot 2024-08-07 at 15 22 44

All the rest LGTM, it is QA ✅ 

Thanks !

**Comment by jolelievre** [comment]
Thanks @tblivet and @florine2623

**Comment by Progi1984** [comment]
@tblivet What is the license of the PrestaFont ?

**Comment by matks** [comment]
> @tblivet What is the license of the PrestaFont ?

The PrestaFont has been included in the open source project by the merge of this PR. As it does not come with a license of its own, then the global project license is applied.

The PrestaFont is now part of the open source project `develop` branch, licensed under the terms of the [The Open Software License 3.0](https://opensource.org/license/osl-3-0-php)

**Pull request merged**
